srcmedia/scss/_mixins.scss

```diff
@@ -180,6 +180,25 @@
             @include small-font-scale;
         }
     }
+
+    blockquote {
+        margin: 2rem 0 2rem 1.5rem;
+        padding-left: 1.5rem;
+        border-left-style: solid;
+        border-left-width: 5px;
+        border-left-color: $pig-pink;
```

Comment:
Seems a little weird to have the specific color here. I don't imagine there's an easy way to have an `$active-color` (or something) that is different in different sections of the site, but I do wonder about using a semantic name instead of the actual color - something like `$about-highlight` or `$editorial-highlight` (or whatever it is semantically, I don't know if it's a highlight color; theme color?).

Reply:
yeah, I absolutely agree - when I refactor this I'll probably either add new variables exactly like that, or change the `@content-page()` mixin to take a color param (or both).